bin/rules_tab.py

```diff
@@ -1171,6 +1178,11 @@ def response_combobox_changed_cb(self, idx):
 
         # if idx == -1:
         #     return
+    #-----------------------------------------------------------
+    def plot_rules(self):
+        RulesWindow = Window_plot_rules()
```

Comment:
`self.RulesWindow` will make it actually display. I don't know why...

Reply:
Yeah! This is one thing that I would like some help.